src/components/confirmation-dialogs.yml

```diff
@@ -73,3 +73,6 @@ closeAccountConfirmation:
 exitFileEditor:
   en: Do you really want to exit the editor? Your current changes will be lost.
   de: Möchtest du die Bearbeitungsansicht wirklich verlassen? Deine Änderungen gehen verloren.
+deleteCommentConfirmation:
+  en: Do you really want to delete this comment from "{author}" posted on "{timestamp}"?
+  de: Möchtest du den Kommentar von "{author}" geschrieben am "{timestamp}" wirklich löschen?
```

Comment:
We need one comma before "geschrieben" and one before "wirklich"

Reply:
We indeed need them, added!